packages/framework/src/Model/Product/Search/Export/ProductSearchExporter.php

```diff
@@ -180,22 +192,4 @@ protected function removeNotUpdated(int $domainId, array $exportedIds): void
     {
         $this->productElasticsearchRepository->deleteNotPresent($domainId, $exportedIds);
     }
-
-    /**
-     * @internal Will be removed in the next major release
-     */
-    protected function validateInjectedDependencies(): void
```

Comment:
Please clarify in the description, that removal of this method is not a BC break as it was introduced in https://github.com/shopsys/shopsys/commit/7919c35af5f8ae499c4dfe6d7ad2b46daebc7302 which hasn't been released yet.

Reply:
I do not think this is necessary

Reply:
Short note in PR description is not a bad idea. 
_Note: removed methods does not cause violation of a BC promise as these methods was not yet part of any released version_

Reply:
Added